contrib/lang/asciidoc/packages.el

```diff
@@ -0,0 +1,41 @@
+;;; packages.el --- Asciidoc Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2015 Mark Safronov, Torben Hoffmann & Contributors
+;;
+;; Author: Mark Safronov <hijarian@gmail.com>
+;; Author: Torben Hoffmann <torben.lehoff@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defvar asciidoc-packages
```

Comment:
Thanks! I forgot about this new requirement since I never use live reloading (I haven't updated my dotfile to work with it). I untagged another PR that I had mistakenly OK'd as well.